pkg/hubble/parser/threefour/parser.go

```diff
@@ -678,7 +680,7 @@ func decodeNetworkInterface(tn *monitor.TraceNotify, dbg *monitor.DebugCapture)
 
 	// if the interface is not found, `name` will be an empty string and thus
 	// omitted in the protobuf message
-	name, _ := link.GetIfNameCached(int(ifIndex))
+	name := p.linkGetter.GetIfNameCached(ifIndex)
```

Comment:
Nit: The old code was intentionally not converting `ifIndex` to a string if the interface wasn't found:

Hubble JSON output before this PR (in case ifIndex refers to a stale/unknown interface):
```json
  "interface": {
    "index": 25,
  }
```

After this PR:
```json
  "interface": {
    "index": 25,
    "name": "25"
  }
```

I believe the `"name": "25"` doesn't provide any value and therefore should be omitted.

Reply:
Ah OK, thanks for the insight. I'll switch this up - I think I'll just revert the `GetIfNameCached` interface change here and maybe put the "always give me a string" version in a central location, maybe just part of the same interface so all the callers who want a string version can get it.

Reply:
I debated just leaving the `ifname.go` file in the monitor package, but figured that once the name getter functions become methods as part of the LinkCache, it makes more sense to just move that code to the structure that it depends on. So in the end I added a new `Name()` method on the interface that allows getting this and always returning a string.